package.json

```diff
@@ -31,7 +31,7 @@
     "coveralls": "^2.11.2",
     "fury": "^2.x.x",
     "peasant": "^0.5.2",
-    "swagger-zoo": "2.2.1"
+    "swagger-zoo": "2.3.x"
```

Comment:
I'd keep this to an exact version. Any changes in the fixtures would break the tests from passing so it's best to pin to the exact version here.